CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.36.1 [2023-10-27]
+
+_Bug fixes_
+
+- Fixed the get call for `github_issue` and `github_pull_request` tables to return data instead of null. ([#355](https://github.com/turbot/steampipe-plugin-github/pull/355))
```

Comment:
```suggestion
- Fixed the `Get Config` of `github_issue` and `github_pull_request` tables to correctly return data instead of an error. ([#355](https://github.com/turbot/steampipe-plugin-github/pull/355))
```